README.md

```diff
@@ -1,7 +1,7 @@
 # About cc65
 
 cc65 is a complete cross development package for 65(C)02 systems, including
-a powerful macro assembler, a C compiler, linker, librarian and several
+a powerful macro assembler, a C compiler, linker, archiver and several
```

Comment:
Was there any consensus on this?
Yes, ar(1) (of gcc) is called "archiver", because it can do more than just a library manager (like LIB.exe from Microsoft).

However, despite its name, ar65 is more like lib.exe than ar(1), because it is not that generic as ar(1), so "librarian" or "library manager" fits much better to it, IMHO.